docs/getting-started/install-endpoint.asciidoc

```diff
@@ -0,0 +1,87 @@
+[[install-endpoint]]
+[role="x-pack"]
+= Configure and install Elastic Security
+
+
+WARNING: The early release of {ingest-management} is experimental. We invite you to
+install and test these capabilities **in a test environment**. You might run
+into problems and need to modify your setup to get this feature running. Please
+do not enable or use {ingest-management} in a production environment.
+
+
+Like other Elastic integrations, Elastic Endpoint Security can be integrated into the Elastic Agent through {ingest-guide}/ingest-management-overview.html[Ingest Manager]. Upon configuration, the integration allows the Elastic Agent to monitor for events on your host and send data to the Elastic Security app.
+
+NOTE: Configuring the Endpoint Integration on the Elastic Agent requires that the user have permission to use Ingest Manager in Kibana.
+
+[discrete]
+[[security-before-you-begin]]
+== Before you begin
+
+If you're using the Elastic Agent on macOS Mojave (10.14) or later, ensure that you have enabled <<sensor-full-disk-access,Full Disk Access>>. Lastly, review the Elastic Security system requirements.
+
+[discrete]
+[[add-security-integration]]
+== Add Elastic Security integration
+
+1. In Kibana, select **Security** > **Administration**. If this is not your first time using Elastic Security, select **Ingest Manager** > **Integrations** and search for "Elastic Endpoint Security".
+2. On the Administration page of the security app or the Elastic Endpoint Security integration page in Ingest Manager, select **Add Endpoint Security**. The integration configuration page appears.
++
+[role="screenshot"]
+image::images/install-endpoint/security-integration.png[]
```

Comment:
This image seems to be more appropriate for step 1 since it displays the integration.  This image shows neither the add button nor the configuration page.  It is not until after you click on the Elastic Endpoint Security panel and then click the 'Add Elastic Endpoint Security' button that the integration configuration page appears.